e2e/windows/e2e-helper.sh

```diff
@@ -16,11 +16,23 @@ exec_on_host() {
     kubectl exec $(kubectl get pod -l app=debug -o jsonpath="{.items[0].metadata.name}") -- bash -c "nsenter -t 1 -m bash -c \"$1\"" > $2
 }
 
-create_storage_account() {
+create_storage_container() {
```

Comment:
What would happen if two pipelines are queued with the same k8s version?

Reply:
Only the first pipeline will create a storage container because `$create_cluster = true`. Other pipelines will only download linux file from storage account. (tested a lot of times) @junjiezhang1997 maybe you can clarify on this?

Reply:
There are three scenarios:
1. resource group and container both do not exist: one pipeline will create the resource group and container, then upload linux files; the other pipeline will wait until linux files are uploaded, then download these files.
2. resource group does not exist, but container exists: one pipeline will create the resource group and clean up files in the container, then upload new linux files; the other pipeline will wait to download files.
3. resource group and container both exist: two pipelines will just go to download linux files from the container and go on testing steps.